examples/vpc_service_control/vpc_sc_project/main.tf

```diff
@@ -0,0 +1,318 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file("${var.service_account_file}")}"
+}
+
+resource "google_project" "vpc_sc_network_project" {
+  name = "VPC SC Network"
+  project_id = "${var.project_id}"
+  org_id     = "${var.organization_id}"
+  billing_account = "${var.billing_account_id}"
+  auto_create_network = false
+}
+
+
+resource "google_compute_network" "test-vpc" {
+  auto_create_subnetworks         = false
+  delete_default_routes_on_create = true
+  name                            = "test-vpc"
+  project                         = "${google_project.vpc_sc_network_project.project_id}"
+  routing_mode                    = "REGIONAL"
+}
+
+
+resource "google_compute_subnetwork" "vpc_subnet" {
+  enable_flow_logs         = false
+  ip_cidr_range            = "10.7.0.0/16"
+  name                     = "vpc-subnet"
+  network                  = "${google_compute_network.test-vpc.self_link}"
+  private_ip_google_access = true
+  project                  = "${google_project.vpc_sc_network_project.project_id}"
+  region                   = "${var.cloud_router_region}"
+}
+
+
+
+
+/* STEP 2A: DELETE THIS COMMENT LINE TO RESERVE STATIC IP (costs ~$0.24/day) 
+resource "google_compute_address" "vpc_sc_vpn_ip" {
+  address_type = "EXTERNAL"
+  name         = "vpc-sc-vpn-ip"
+  network_tier = "PREMIUM"
+  project      = "${google_project.vpc_sc_network_project.project_id}"
+  region       = "${var.cloud_router_region}"
+}
+
+output "ip_addr_of_cloud_vpn_router" {
```

Comment:
All outputs should be in a `outputs.tf` file not in the `main.tf` file.

Reply:
I can't put it in outputs.tf because it's part of the commented-out block of code.  If I put it in outputs.tf, that's a new file where users will have to deal with commenting and uncommenting.  Makes it more difficult to use.

Reply:
I understand your problem. Though, I think this is another example of the commenting/uncommenting workflow leading to bad Terraform practices. So again, I suggest changing the workflow and apply Terraform best practices as mentioned [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/pull/4#discussion_r295591024)